doctr/transforms/modules/tensorflow.py

```diff
@@ -375,3 +376,44 @@ def __call__(self, x: tf.Tensor) -> tf.Tensor:
 
     def extra_repr(self) -> str:
         return f"mean={self.mean}, std={self.std}"
+
+
+class RandomHorizontalFlip(NestedObject):
+    """Adds random horizontal flip to the input tensor/np.ndarray
+
+    Example::
+           >>> from doctr.transforms import RandomHorizontalFlip
+           >>> import tensorflow as tf
+           >>> transfo = RandomHorizontalFlip(p=0.5)
+           >>> image = tf.random.uniform(shape=[64, 64, 3], minval=0, maxval=1)
+           >>> target = {
+            "boxes": np.array([[0.1, 0.1, 0.4, 0.5] ], dtype= np.float32),
+            "labels": np.ones(1, dtype= np.int64)
+            }
+           >>> out = transfo(image, target)
+
+       Args:
+           p : probability of Horizontal Flip"""
+    def __init__(self, p):
+        super().__init__()
+        self.p = p
+
+    def __call__(
+            self,
+            img: Union[tf.Tensor, np.ndarray],
```

Comment:
? are we positive that this works with `np.ndarray`?

Reply:
Yes, it works with numpy array as well :)

Reply:
Yes, I checked it works with `np.ndarray` as well.

Reply:
And the return type is preserved? (when sending a `np.ndarray`, does it return a `np.ndarray`?)

Reply:
Thanks for the correction. No, the type is not preserved. It returns a tf.tensor. Thanks!